app/src/app/components/login/BasicUserAuthForm.tsx

```diff
@@ -0,0 +1,87 @@
+import { zodResolver } from "@hookform/resolvers/zod";
+import { Mail } from "lucide-react";
+import { useForm } from "react-hook-form";
+import { useNavigate } from "react-router-dom";
+import { z } from "zod";
+import appConfig from "../../../config/appConfig";
+import { ApiError } from "../../../lib/errors";
+import { fetcher } from "../../../lib/fetch";
+import { Button } from "../Base/Button";
+import { Form, FormControl, FormField, FormItem, FormLabel, FormMessage } from "../Base/Form";
+import { Input } from "../Base/Input";
+import { useUser } from "../providers/UserProvider";
+
+export const BasicUserAuthForm = () => {
+  const navigate = useNavigate();
+  const { setUser } = useUser();
+
+  const formSchema = z.object({
+    email: z.string().email(),
+    password: z.string().min(8)
```

Comment:
Is this saying the password needs to be at least 8 characters?
That doesn't seem like something we should be enforcing in the login form, but rather when credentials are created.

Reply:
its still a form validation so we dont send the request ... ive added at least 8 character password when creating users in the future tickets